libraries/hermes/src/Proxy/Proxy.ts

```diff
@@ -0,0 +1,89 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import { ConduitRouter } from '../Router';
+import ConduitGrpcSdk from '@conduitplatform/grpc-sdk';
+import { createProxyMiddleware } from 'http-proxy-middleware';
+import { ProxyRoute, TypeRegistry } from '../classes';
+
+export class ProxyRouteController extends ConduitRouter {
+  private _proxyRoutes: Map<string, ProxyRoute>;
+  private globalMiddlewares: ((
+    req: Request,
+    res: Response,
+    next: NextFunction,
+  ) => void)[];
+
+  constructor(
+    grpcSdk: ConduitGrpcSdk,
+    private readonly metrics?: {
+      registeredRoutes?: {
+        name: string;
+      };
+    },
+  ) {
+    super(grpcSdk);
+    this._proxyRoutes = new Map();
+    this.globalMiddlewares = [];
+    this.initializeRouter();
+  }
+
+  private initializeRouter() {
+    this.createRouter();
+    this._expressRouter!.use((req: Request, res: Response, next: NextFunction) => {
+      this.globalMiddlewares.forEach(middleware => middleware(req, res, next));
+    });
+  }
+
+  registerGlobalMiddleware(
```

Comment:
Why is this here? The proxy router is not like the socket one, it uses the global express router that already handles global middlewares